docs/examples/avatars.js

```diff
@@ -26,5 +23,5 @@ client.on('message', message => {
   }
 });
 
-// Log our bot in
-client.login(token);
```

Comment:
useless template string, same almost every string you changed

Reply:
Ooppss.. Too used to using ` in strings.